lib/player.rb

```diff
@@ -50,5 +51,24 @@ def highest_word_score
       highest_scoring_word = ScrabbleMod::Scrabble.highest_score_from(@plays)
```

Comment:
In the `won?` method above, should it use the `@won` instance variable? I'm wondering what this is used for?